runner/run_test.go

```diff
@@ -547,16 +555,17 @@ func TestBrokenPortOverrideRun(t *testing.T) {
 	replaceDestinationInTest(&ftwTest, *fakeDestination)
 
 	// the test should succeed, despite the unknown override property
-	res := Run([]test.FTWTest{ftwTest}, Config{
+	res, err := Run([]test.FTWTest{ftwTest}, Config{
 		Quiet: true,
 	})
+	assert.NoError(t, err)
 	assert.LessOrEqual(t, 0, res.Stats.TotalFailed(), "Oops, test run failed!")
 }
 
 func TestDisabledRun(t *testing.T) {
 	t.Cleanup(config.Reset)
 
-	err := config.NewConfigFromString(yamlConfig)
+	err := config.NewConfigFromString(yamlCloudConfig)
```

Comment:
Why did you have to change the config here?

Reply:
Thanks for checking forgot to mention it previously the presence of a log file was checked lazily so if we the test didn't run, like this one checking behavior of disabled tests, it was ok without a log file set. Now that validation is eager, which should generally be good. It seemed simplest to use cloud mode here which doesn't need a log file instead of setting up an unused file